src/metabase/notification/payload/execute.clj

```diff
@@ -0,0 +1,191 @@
+(ns metabase.notification.payload.execute
+  (:require
+   [malli.core :as mc]
+   [metabase.api.common :as api]
+   [metabase.models.dashboard-card :as dashboard-card]
+   [metabase.models.interface :as mi]
+   [metabase.models.params.shared :as shared.params]
+   [metabase.models.serialization :as serdes]
+   #_{:clj-kondo/ignore [:metabase/ns-module-checker]}
```

Comment:
yea the `metabase.pulse` is a module with public apis, ideally we should import `metabase.pulse` and use the exported functions there. But doing that will create a cyclic dep so I avoided it here.

I think it's ok because we'll move those stuffs to notification.